src/cards.py

```diff
@@ -0,0 +1,29 @@
+
+
+def sort_cards(cards):
+    """Sort a deck of cards."""
+    deck = []
+    new_deck = []
+    for card in cards:
+        if card.upper() == 'A':
+            deck.append((0, 'A'))
+
+        elif card.upper() == 'T':
+            deck.append((10, 'T'))
+
+        elif card.upper() == 'J':
+            deck.append((11, 'J'))
+
+        elif card.upper() == 'Q':
+            deck.append((12, 'Q'))
+
+        elif card.upper() == 'K':
+            deck.append((13, 'K'))
+
+        else:
+            deck.append((int(card), card))
+    deck.sort()
+    for card in deck:
+        card = card[1]
+        new_deck.append(card)
+    return new_deck
```

Comment:
Consider this solution instead, Mike:

```python
def sort_cards(cards):
    values = {"A": 1, "2": 2, "3": 3, "4": 4, '5': 5, '6': 6, '7': 7, '8': 8, '9': 9, 'T': 10, 'J': 11, 'Q': 12, 'K': 13}    
    prioritized = sorted([(card, values[card]) for card in cards], key=lambda x: x[1])
    return [card[0] for card in prioritized]
```

Basically doing the exact same thing you're doing, except not going through a whole control flow to do it. You're sending every card in the given list to a new list and then to a third list once sorted. Why do all that? Practice using lambda.